core/plugins/jdo-datanucleus-5/src/main/java/org/apache/isis/core/runtime/system/persistence/PersistenceSession5_Decouple.java

```diff
@@ -82,7 +82,13 @@ protected PersistenceSession5_Decouple(PersistenceSession5 holder, ObjectAdapter
     public ObjectAdapter adapterFor(
             final RootOid rootOid,
             final ConcurrencyChecking concurrencyChecking) {
-
+        
+        //FIXME[ISIS-1976] guard against service lookup
+        final ObjectAdapter serviceAdapter = objectAdapterContext.lookupServiceAdapterFor(rootOid);
+        if (serviceAdapter != null) {
+            return serviceAdapter;
+        }
+        
```

Comment:
this whole class is regarded temporary ... refectoring in progress

Reply:
yeah, I figured.  Fine with me